### DISCUSSION TITLE
Pause during play

### DISCUSSION THREAD

**Discussion by AN0NCER** [ppy/osu#22066] [Q&A] [answered]
I faced such problem, when I play osu and the map lasts 4 minutes, I need to grab the handle. But the pause that is on the map is 1 second, and I press ESC and I do not pause. Only when I hold down ESC only then will it pause, but during this time my 1 second will pass. How can I remove the pause delay?

**Comment by vioco** [comment, marked as answer]
You can search hold-to-confirm activation time and change it to 0 ms.
